docs/Demo.md

```diff
@@ -1,447 +1,409 @@
 * **Deployment Frequency:**
     * Deployment Frequency = {number of deploys in a defined time frame}
+    * There have been two deployments since this demonstration was started, the initial deployment and now the redeployment after pushing a change to the git repository. The deployment frequency should have gone up by 100% in the last 15 minutes. Once the initial deployment time is longer than 15 minutes in the past the interval will fall by 50%.
 
-    * There have been two deployments since this demonstration was started, the initial deployment and now the redeployment after pushing a change to the git repository.  The deployment frequency should have gone up by 100% in the last 15 minutes.  Once your initial deployment time is longer than 15 minutes in the past, you will find your interval has fallen by 50%.
+**Procedure**
+1. Get the URL to navigate to Grafana.
+```
+oc -n pelorus get route grafana-route -o json | jq -r '.spec.host'
+```
+1. Click **Home** at the top left of the screen, **Pelorus**, then **Software Delivery Performance**.
+![First-Update](img/todolist_update1.png)
 
-### See the raw data in the Pelorus Exporter logs
-Check the Pelorus committime output for the commit hash that was pushed:
+The raw data can now be viewed in the Pelorus Exporter logs.
 
+3. Check the Pelorus committime output for the commit hash that was pushed.
 ```
 git log -p -1
 ```
-
-Compare the output with the following command:
-
+4. Compare the output with the following command.
 ```
 curl $(oc get route -n pelorus committime-exporter -o=template='http://{{.spec.host | printf "%s\n"}}')
 ```
-
-Also notice the built deployed image sha is visible via the deploytime-exporter
-
+5. View the built deployed image sha using the deploytime-exporter.
 ```
 curl $(oc get route -n pelorus deploytime-exporter -o=template='http://{{.spec.host | printf "%s\n"}}')
 ```
-
-### See the change to your todolist application
-You can now also check on your todolist application and see the updated text change "Add a todo"
+6. Check the todolist application and verify the text has changed to "Add a todo".
 
 ![todolist-fixed](img/todolist_fixed.png)
 
 
 ## Troubleshooting
-
-If exporters are not functioning or deployed, no data will show up in the dashboard. It will look like the following:
+If the exporters are not deployed or functioning, no data will show up in the dashboard.
 
 ![No-Data](img/pelorus-dashboard-no-data.png)
 
-* Please check the logs of exporter pod.
-
-An "idle" state could resemble:
+If this occurs, check the exporter pod logs. An idle state might look like the following:
 
 ![Idle-Data](img/pelorus-dashboard-idle-data.png)
 
 
-# Mean Time to Restore and Change Failure Rate
+## Setting Mean Time to Restore and Change Failure Rate
```

Comment:
please remove the word "Setting"  it is intentionally "Mean Time to Restore and Change Failure Rate"